Readme.md

```diff
@@ -4,19 +4,72 @@
 [![NPM Downloads][downloads-image]][downloads-url]
 [![Build Status][travis-image]][travis-url]
 
-Utility to parse a string bytes (ex: `1TB`) to bytes (`1099511627776`) and vice-versa.
+Utility to parse a string bytes (ex: `1TB`) to bytes (`1,000,000,000,000`) and vice-versa.
+
+This uses the byte units defined in ISO/IEC 80000-13:2008, both the binary prefixes and the original SI units.
+
+Supported units and abbreviations are as follows and are case-insensitive:
+
+### Metric/Decimal Prefixes
+
+|      Value       | Abbr |   Name    |
+| ---------------- | ---- | --------- |
+| 1                | B    | byte      |
```

Comment:
Why not `1000<sup>0</sup>` to keep with the same pattern? Also, using HTML is hard to read in text editors, so I would say if you can not use HTML in the MD, that would be best...